.snippets/code/develop/parachain-devs/interoperability/XCM/XCM-first-look.rs

```diff
@@ -0,0 +1,17 @@
+let message = Xcm(vec![
+    WithdrawAsset((Here, amount).into()),
+    BuyExecution { 
+        fees: (Here, amount).into(), 
+        weight_limit: WeightLimit::Unlimited 
+    },
+    DepositAsset {
+        assets: All.into(),
+        beneficiary: MultiLocation {
+            parents: 0,
+            interior: Junction::AccountId32 {
+                network: None,
+                id: BOB.clone().into()
+            }.into(),
```

Comment:
Is this .into() required? If not, maybe we can remove it so the code looks even cleaner wdyt?

Reply:
@RemyLeBerre, where does this example come from? I just wanted to double-check the context here